Makefile

```diff
@@ -87,6 +87,10 @@ KUBEADM_BOOTSTRAP_CONTROLLER_IMG ?= $(REGISTRY)/$(KUBEADM_BOOTSTRAP_IMAGE_NAME)
 KUBEADM_CONTROL_PLANE_IMAGE_NAME ?= kubeadm-control-plane-controller
 KUBEADM_CONTROL_PLANE_CONTROLLER_IMG ?= $(REGISTRY)/$(KUBEADM_CONTROL_PLANE_IMAGE_NAME)
 
+# operator
```

Comment:
background question:
given this is an experiment, should we host those targets in the main Makefile or have a separated Makefile like we are doing for CAPD?

Reply:
yes, this is a good suggestion. I'm happy to do this in a follow-up PR.

Reply:
Could we have those changes in this PR? It seems appropriate given that we're just introducing this.